scrapy/utils/project.py

```diff
@@ -70,6 +71,9 @@ def get_project_settings():
     # XXX: remove this hack
     pickled_settings = os.environ.get("SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE")
     if pickled_settings:
+        warnings.warn("Use of environment variable "
+                      "'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE'"
+                      " is deprecated.",ScrapyDeprecationWarning)
```

Comment:
To follow PEP8 it's necessary to add a whitespace after the comma (that means between: `" is deprecated.",` and `ScrapyDeprecationWarning`)

```suggestion
                      " is deprecated.", ScrapyDeprecationWarning)
```

Reply:
Already suggested by Gallaecio, resolving :)